src/os/bluestore/BlueStore.cc

```diff
@@ -7441,6 +7441,12 @@ int BlueStore::expand_devices(ostream& out)
           << std::endl;
       }
     }
+
```

Comment:
Better doing all the below stuff under the "if (size0 < size) {}' clause above?

Reply:
Not sure what you mean - everything was done under the 'if (size0 < size) {}' clause..